IceCreamShop.playground/Contents.swift

```diff
@@ -1 +1,95 @@
 
+struct Flavor {
+    let name: String
+    let rating: Int
+}
+
+enum Size: Double {
+    case kidsCup = 2.49
+    case smallCup = 2.99
+    case mediumCup = 3.99
+    case largeCup = 4.99
+    case kidsCone = 2.69
+    case smallCone = 3.09
+    case mediumCone = 4.09
+    case largeCone = 5.09
+    case waffleCone = 5.49
+}
+
+struct Cone {
+    let flavor: Flavor
+    let topping: String
```

Comment:
Keep in mind that they may or may not want a topping. So what should we do with this?